pkg/cmd/traces.go

```diff
@@ -0,0 +1,119 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/go-kit/log/level"
+	"github.com/observatorium/obsctl/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+func NewTraceServicesCmd(ctx context.Context) *cobra.Command {
+	var outputFormat string
+	cmd := &cobra.Command{
+		Use:   "services",
+		Short: "List names of services",
+		Long:  "List names of services with trace information",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Don't print CLI flag usage if we get network error
+			cmd.SilenceUsage = true
+
+			cfg, err := config.Read(logger)
+			if err != nil {
+				return fmt.Errorf("getting reading config: %w", err)
+			}
+
+			client, err := cfg.Client(ctx, logger)
+			if err != nil {
+				return fmt.Errorf("getting current client: %w", err)
+			}
+
+			level.Debug(logger).Log(
+				"msg", "Using configuration",
+				"URL", cfg.APIs[cfg.Current.API].URL,
+				"tenant", cfg.Current.Tenant)
+
+			url := fmt.Sprintf("%s%s", cfg.APIs[cfg.Current.API].URL, "api/traces/v1/rhobs/api/services")
```

Comment:
nit: I like to use `path.Join` for these kind of things to make sure the slashes are always in place.

Reply:
Agreed.  The problem is that `cfg.APIs[cfg.Current.API].URL` has the value `"https://observatorium.apps.observability-d.p3ao.p1.openshiftapps.com/"`.

`path.Join() would` have replaced `https://` with `https:/`.  Cleaner code:

```Golang
			svcUrl, err := url.Parse(cfg.APIs[cfg.Current.API].URL)
			if err != nil {
				return fmt.Errorf("parsing url: %w", err)
			}
			svcUrl.Path = "api/traces/v1/rhobs/api/services"
			resp, err := client.Get(svcUrl.String())
```